app/styles/ui/_text-box.scss

```
@@ -24,6 +24,10 @@
     margin-bottom: var(--spacing-third);
   }
 
+  :invalid{
```

Missing a little space here to conform to our conventions

```suggestion
  :invalid {
```